main.tf

```diff
@@ -1,22 +1,10 @@
-terraform {
-  required_providers {
-    aws = {
-      source  = "hashicorp/aws"
-      version = "~> 4.0"
-    }
-  }
-}
 
-# Configure the AWS Provider
-provider "aws" {
-  region = "us-east-1"
-}
 
-resource "aws_instance" "app_server" {
-  ami           = "ami-02396cdd13e9a1257"
-  instance_type = "t3.nano"
+# resource "aws_instance" "app_server" {
+#   ami           = "ami-02396cdd13e9a1257"
+#   instance_type = "t3.nano"
 
-  tags = {
-    Name = "ExampleAppServerInstance"
-  }
-}
+#   tags = {
+#     Name = "ExampleAppServerInstance"
+#   }
+# }
```

Comment:
Please don't leave commented-out code blocks. Instead, remove them completely.

Reply:
Thank you, I've deleted the commented code.